README.md

```diff
@@ -41,229 +42,300 @@ PID   USER     COMMAND
 / $
 ```
 
-Or you can specify the path to a JSON configuration file:
-```bash
-runc start config.json
-/ $ ps
-PID   USER     COMMAND
-1     daemon   sh
-5     daemon   sh
-/ $
-```
-Note: the use of the `start` command is required when specifying a 
-configuration file.
+### OCI Container JSON Format:
 
-### OCF Container JSON Format:
-
-Below is a sample `config.json` configuration file. It assumes that
+Below are sample `config.json` and `runtime.json` configuration files. It assumes that
```

Comment:
Agree, let's do this in another PR though. It was supposed to be mostly code PR to get runc working.